Beam/Config/Python/CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ endif()
 if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX OR
     ${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fvisibility=hidden")
+  if(${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
```

Comment:
May as well enable this for GCC as well instead of just Clang, also no need to specify it for ```CMAKE_CXX_FLAGS``` and ```CMAKE_CXX_FLAGS_RELEASE``` since the former includes the latter.

Reply:
Should have no effect on GCC, but makes it less verbose.